feat(serverless): add PN info (#3739)
scaleway/docs-content: faq/serverless-jobs.mdx

## Do Serverless Jobs offer parallelization?

Not yet. Scaleway Serverless Jobs will soon offer parallelization via a `parallelism` parameter for each job. This will automatically launch a given number of replicas of the Job container in parallel.

> ## How to configure access to a Private Network ?

Reviewer: ```suggestion
## How can I configure access to a Private Network?
```

> Scaleway Serverless Jobs does not support Virtual Private Networks (VPC) for now, it's a feature under development.

Reviewer: ```suggestion
Scaleway Serverless Jobs does not currently support Scaleway VPC or Private Networks, though this feature is under development.
```

> If it's possible to add some network restrictions on your resource, here is the list of prefixes used at Scaleway: [https://www.scaleway.com/en/peering/](https://www.scaleway.com/en/peering/). Serverless ressources does not have dedicated and predictible IP address.

Reviewer: ```suggestion
To add network restrictions on your resource, consult the [list of prefixes used at Scaleway](https://www.scaleway.com/en/peering/). Note that Serverless resources do not have dedicated or predictable IP addresses.
```
